src/controllers/diff.controller.ts

```diff
@@ -0,0 +1,65 @@
+import { NextFunction, Request, Response, Router } from 'express';
+import Ajv from 'ajv';
+import { ProblemException } from '../exceptions/problem.exception';
+import { Controller} from '../interfaces';
+
+import { documentValidationMiddleware } from '../middlewares/document-validation.middleware';
+import { DiffService } from '../services/diff.service';
+
+
+export class DiffController implements Controller {
+    public basepath = '/diff';
+    private ajv: Ajv;
+    diffService: DiffService;
```

Comment:
You don't need the `ajv`. We validate the request body in the https://github.com/asyncapi/server-api/blob/master/src/app.ts#L58 middleware which runs on every request. Also `diffService` is unnecessary because it's only single method, so you don't need to wrap the `diff` library in the separate service.